webots_ros2_driver/src/PythonPlugin.cpp

```diff
@@ -29,17 +29,31 @@ namespace webots_ros2_driver
 
         PyObject *pyWebotsExtraModuleSource = Py_CompileString(
             R"EOT(
-from webots_ros2_driver_webots.controller import Supervisor
+import sys, os
+import webots_ros2_driver_webots
+
+path = os.path.dirname(webots_ros2_driver_webots.__file__)
+
+sys.path.insert(1, path)
+
+from vehicle import Driver
```

Comment:
```suggestion
sys.path.insert(0, os.path.dirname(webots_ros2_driver_webots.__file__))
from vehicle import Driver
```
Avoid exposing unnecessary variables and group related instructions. Also, why `1` and not `0`?

Reply:
Alright for exposing !

I put `1` instead of `0` because I read that the position `0` is risky some times.